### DISCUSSION TITLE
Multiple directus files translations for same language and field

### DISCUSSION THREAD

**Discussion by AlessiaCorsini** [directus/directus#13663] [Archive]
I am having some problems with the directus files translations.
In the directus files model I have added a new translation field 'caption'.  
When editing the file entry  I can edit /save the field per each locale but if edit it a second time and I reopen the entry the content for the field is missing.

If I look via api I can see that the translation field contains multiple junctions for the same field and language

`
{
  translations: [
    {
      id: 24,
      directus_files_id: "6d293642-729a-4bc3-8b57-2e992a35f764",
      languages_code: "de-DE",
      caption: "<p>Test 1</p>",
    },
    {
      id: 175,
      directus_files_id: "6d293642-729a-4bc3-8b57-2e992a35f764",
      languages_code: "de-DE",
      caption: "<p>Test 2</p>",
    },
    {
      id: 176,
      directus_files_id: "6d293642-729a-4bc3-8b57-2e992a35f764",
      languages_code: "en-US",
      caption: "<p>Test 3</p>",
    },
    {
      id: 346,
      directus_files_id: "6d293642-729a-4bc3-8b57-2e992a35f764",
      languages_code: "de-DE",
      caption: "<p>Test 4</p>",
    },
  ],
}
`
It seems that instead of editing the existing junction it creates a new one.
Any ideas on how to fix it?  Thank you!
We started this project with Directus 9.6 and now update it to 9.11.1

**Comment by rijkvanzanten** [comment]
@Nitwel Could this be from the relational interface using the `/items/` endpoint for a check, instead of `/files`? 🤔
